kawa/InputSourceManager.swift

```diff
@@ -36,6 +37,9 @@ class InputSource: Equatable {
         id = InputSource.getProperty(tisInputSource, kTISPropertyInputSourceID)!
         name = InputSource.getProperty(tisInputSource, kTISPropertyLocalizedName)!
 
+        let langs: Array<String> = InputSource.getProperty(tisInputSource, kTISPropertyInputSourceLanguages)!
+        isCJK = langs.contains(where: { $0 == "ko" || $0 == "ja" || $0.hasPrefix("zh") })
```

Comment:
I don't know much about that bug, but I saw somewhere that it's "CJKV-related", so _probably_ Vietnamese also deserves to be here.. Or if it's about all complicated input sources, _probably_ Arabic and Hebrew should also be here (just guessing).